File: tema_pclp_c_lab4/ex3.c
#include <stdio.h>
int main()
{
    int N;
    printf("Va rog introduceti un numar: ");
    scanf("%d", &N);
    int a[N][N];
    for (int i = 0; i < N; i++)
        for (int j = 0; j < N; j++)
        {
            if (j >= i)
                a[i][j] = i + j + 1;
            else
                a[i][j] = a[j][i];
        }
    for (int i = 0; i < N; i++) {
        for (int j = 0; j < N; j++)
            printf("%d ", a[i][j]);
        printf("\n");
    }

    return 0;
}

File: Java/tema4/ex3.java
package tema4;

import java.util.Scanner;

public class facturaServiciu {
	private String denumireServiciu;
	private double tarifOra;
	private double numarOre;
	private double valoareTotala;
	public double valoareTotala(double tarifOra, double numarOre)
	{
		valoareTotala=tarifOra*numarOre;
		return valoareTotala;
	}
	public facturaServiciu(String denumireServiciu, double tarifOra, double numarOre, double valoareTotala)
	{
		this.denumireServiciu=denumireServiciu;
		this.tarifOra=tarifOra;
		this.numarOre=numarOre;
		this.valoareTotala=valoareTotala(tarifOra, numarOre);
	}
	public void afisareFactura()
	{
		System.out.println("Denumire serviciu: "+denumireServiciu);
		System.out.println("Tarif orar: "+tarifOra);
		System.out.println("Numar ore: "+numarOre);
		System.out.println("Valoare Totala: "+valoareTotala);
	}
	public static void main(String[] args)
	{
		Scanner s=new Scanner(System.in);
		System.out.println("Introduceti denumirea serviciului: ");
		String denumireServiciu=s.nextLine();
		System.out.println("Introduceti tariful: ");
		double tarifOra=s.nextDouble();
		System.out.println("Introduceti numar ore: ");
		double numarOre=s.nextDouble();
		double valoareTotala=0;
		facturaServiciu f1=new facturaServiciu(denumireServiciu, tarifOra, numarOre, valoareTotala);
		f1.afisareFactura();
	}
}
